src/consensus/vote.rs

```diff
@@ -33,13 +33,10 @@ pub(crate) enum Vote {
     // Voted to update the elders info of a section.
     SectionInfo(EldersInfo),
 
-    // Voted to update our section key.
-    OurKey {
-        // In case of split, this prefix is used to differentiate the subsections. It's not part of
-        // the proof and thus not serialised when signing.
-        prefix: Prefix,
-        key: bls::PublicKey,
-    },
+    // Voted to update the elders in our section.
+    // NOTE: the `EldersInfo` is already signed with the new key. All that's left to do is to sign
+    // the new key using our current key, which is what this vote is for.
```

Comment:
No, what's signed is only the key, but the vote contains the other stuff so that when it reaches consensus, we can update the new elders info and the new key at the same time. I'll try to update the comment to make is more clear.